game_env.py

```diff
@@ -897,6 +897,73 @@ def play(self):
 
         return winner
 
+    def create_board_reps(self, transition_list):
+        """
+        Returns a dictionary containing transition list of each transformation of the board
+        Total of 12 representation are possible 4 sides of the board X 3 views - normal, horizontal-flip
+        vertical - flip
+        # takes 3ms per transition list
+
+        Returns
+        --------
+        transition_dict - Dictionary consisting of transition list of each representation of the board  
+
+        """
+
+        s, legal_moves, current_player, a, next_s,\
+        next_legal_moves, next_player, done, winner = transition_list
+        
+        s1_nd = self._converter.convert(s[0], input_format='bitboard_single', output_format='ndarray')
+        s2_nd = self._converter.convert(s[1], input_format='bitboard_single', output_format='ndarray')
+        legal_moves_nd = self._converter.convert(legal_moves, input_format='bitboard_single',\
+                                                 output_format='ndarray')
+        a_nd = self._converter.convert(a, input_format='bitboard_single', output_format='ndarray')
+        next_s1_nd = self._converter.convert(next_s[0], input_format='bitboard_single',\
+                                             output_format='ndarray')
+        next_s2_nd = self._converter.convert(next_s[1], input_format='bitboard_single',\
+                                             output_format='ndarray')
+        next_legal_moves_nd = self._converter.convert(next_legal_moves, input_format='bitboard_single',\
+                                                      output_format='ndarray')
+        nd_list = [s1_nd, s2_nd, legal_moves_nd, a_nd, next_s1_nd, next_s2_nd, next_legal_moves_nd]
+
+        transition_dict = {}
+        transition_dict['normal_bottom'] = transition_list
+
+        self._nd_trans_fn = {'normal_left': lambda x: np.rot90(x, 1),
```

Comment:
try bitboard conversions from here
https://www.chessprogramming.org/Flipping_Mirroring_and_Rotating

Reply:
Also add diagonal and anti diagonal filps here, after all the transitions, we also need to see which of this modifications are resulting in the same board, we might be able to avoid some operations this way